app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    @abstractmethod

```

Comment:
The `@abstractmethod` decorator is not appropriate for the `__init__` method. Constructors should not be abstract because they are meant to initialize instances of the class. Consider removing the `@abstractmethod` decorator from the `__init__` method.